packages/react-vapor/src/utils/DateUtils.ts

```diff
@@ -72,12 +72,14 @@ export class DateUtils {
     }
 
     static getValidDate(date: string, fromTime: boolean = false): Date {
-        const momentDate: moment.Moment = moment(date, LONG_DATE_FORMAT, fromTime);
-        if (momentDate.isValid()) {
-            return momentDate.toDate();
-        }
+        if (date !== '') {

```

Comment:
Shouldn't that empty string check be done in the DatePicker component instead? In the scope of the `getValidDate` static method, it makes sens to me to return `now` when getting empty string